macosx/TorrentTableView.mm

```diff
@@ -667,11 +667,16 @@ - (void)paste:(id)sender
     }
     else
     {
-        NSArray* items = [NSPasteboard.generalPasteboard readObjectsForClasses:@[ [NSString class] ] options:nil];
-        if (items)
+        NSArray<NSString*>* items = [NSPasteboard.generalPasteboard readObjectsForClasses:@[ [NSString class] ] options:nil];
+        if (!items)
         {
-            NSDataDetector* detector = [NSDataDetector dataDetectorWithTypes:NSTextCheckingTypeLink error:nil];
-            for (__strong NSString* pbItem in items)
+            return;
+        }
+        NSDataDetector* detector = [NSDataDetector dataDetectorWithTypes:NSTextCheckingTypeLink error:nil];
+        for (NSString* itemString in items)
+        {
+            NSArray<NSString*>* itemLines = [itemString componentsSeparatedByCharactersInSet:NSCharacterSet.newlineCharacterSet];
+            for (__strong NSString* pbItem in itemLines)
```

Comment:
`__strong` is not needed as the reference is strong by default

Reply:
It is needed explicitly in ARC's fast enumeration when the object is re-assigned, which is the case two lines below.
Try to remove `__strong` and it won't compile anymore.

Reply:
Right, I thought it was an artefact from the MRR-ARC conversion of years ago. LGTM